wasm-engines/build_bench_images.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+set -e
+
+# (cd docker && \
```

Comment:
Are these comments needed?

Reply:
this file isn't used.  going to remove it.